index.html

```diff
@@ -27,7 +27,11 @@
             <li>Contact</li>
         </nav>
         
-        <div class="middle-image">
+        <div class="middle-image" style="
+                                         display:flex;
+                                         justify-content: center;
+                                         align-items: center;
+                                         ">
             <img src="img/jumbo.jpg" alt="Full width elapsed time night traffic image">
         </div>
     </header>
```

Comment:
remember to link all the nav items in both header and footer in all pages so you can easily navigate between them.
* in the self study questions on the first one, keep in mind that semantic HTML is not a methodology but a whole new set of symantic tags that were added into html in html5. they give more meaning to screen readers and search engine bots for indexing and sectioning your page, both for SEO, search results, and navigating through it with the screen reader for those that are visually impaired.
* you can utilize the cursor:pointer property on your buttons on the about page to help indicate to the users that the buttons are intended to be clicked
* good job overall, i know the week was challenging with the computer issues, but you where able to pull through